_qs/200-workspace.md

```diff
@@ -65,6 +65,7 @@ So click `Next`, which will bring us to a page asking us if we should import the
 
 ![Import archetype](/img/qs/git-import-6.png)
 
+<!-- I don't understand the next sentence. -->
```

Comment:
This sort of comment is hard for another to understand when reading in the future. No one will know that I == Grant. :-)